playbooks/provisioning/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -4,7 +4,7 @@ public_dns_domain: "example.com"
 public_dns_nameservers: []
 
 openstack_ssh_public_key: "openshift"
-openstack_default_image_name: "rhel73"
+openstack_default_image_name: "centos7"
 openstack_default_flavor: "m1.medium"
```

Comment:
Yeah, though I don't want to disable these checks in the sample inventory. This is something people trying it out should be made aware of.

The ci script adds this configuration explicitly.